src/main/java/org/alliancegenome/curation_api/model/ingest/dto/fms/UberonSlimTermDTO.java

```diff
@@ -0,0 +1,11 @@
+package org.alliancegenome.curation_api.model.ingest.dto.fms;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.model.ingest.dto.base.BaseDTO;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class UberonSlimTermDTO extends BaseDTO {
```

Comment:
This class name should be `UberonStageSlimTerm` from the schema repo?

Reply:
Ahh maybe the Structure and Stage terms are being combined into this one?

Reply:
Yes, they have the same format in the DTO